docs/deployment/engine_on_kubernetes.md

```diff
@@ -48,6 +48,12 @@ The minimum required configurations are:
 * spark.kubernetes.file.upload.path (path on S3 or HDFS)
 * spark.kubernetes.authenticate.driver.serviceAccountName ([viz ServiceAccount](#serviceaccount))
 
+The vanilla Spark neither support rolling nor expiration mechanism for `spark.kubernetes.file.upload.path`, if you use
+file system that does not support TTL, e.g. HDFS, additional cleanup mechanisms are needed to prevent the files in this
+directory from growing indefinitely. Since Kyuubi v1.11.0, you can configure `spark.kubernetes.file.upload.path` with
+placeholders `{{YEAR}}`, `{{MONTH}}` and `{{DAY}}`, and enable `kyuubi.kubernetes.spark.autoCreateFileUploadPath.enabled`
+to let Kyuubi server create the directory with 777 permission automatically before submitting Spark application.
+
```

Comment:
It seems that our current implementation does not solve the problem of file growth. Will this issue be solved in subsequent PRs?

Reply:
It adds the rolling support for `spark.kubernetes.file.upload.path`, for example,

```
spark.kubernetes.file.upload.path=hdfs://hadoop-testing/spark-upload-{{YEAR}}{{MONTH}}
```

```
hdfs://hadoop-testing/spark-upload-202412
hdfs://hadoop-testing/spark-upload-202501
```
Admin can safely delete the `hdfs://hadoop-testing/spark-upload-202412` after 20250101